test/TEST-63-DRACUT-CPIO/test.sh

```diff
@@ -24,16 +24,15 @@ test_dracut_cpio() {
     # VM script to print sentinel on boot
     # write to kmsg so that sysrq messages don't race with console output
```

Comment:
you can probably get rid of the above line, although won't the race still be a problem with poweroff triggering the shutdown instead of sysrq?